tools/ktx/command_help.cpp

```diff
@@ -161,15 +164,15 @@ void CommandHelp::executeHelp() {
         fatal(rc::RUNTIME_ERROR, "Failed to open the html documentation: ERROR {}", r);
 
 #else
-#   if defined(__APPLE__)
+#if defined(__APPLE__)
```

Comment:
I prefer the indenting. Maybe use `#elif`.

Reply:
Yeah, it's from an old code, I'll update it.

Reply:
Wait, you're wrong about this one, as Apple and Linux platform shares most of the code, so `#elif` and a separate partially nested `#ifdef` is the correct solution.